Automattic/pocket-casts-ios
# Insert a note to the support ticket to indicate logs are attached

| 📘 Part of: # |  
|:---:|

Fixes # 


Adds code to Insert a note to the support ticket to indicate logs are attached  to make automation better on server side

## To test

1. Start the app
2. Go to Profile -> Help & Feedback
3. Scroll to the bottom and tap Get in Touch
4. Tap on Get Support
5. Fill the support form
6. And ensure you have logs attached
7. Send the request
8. Check on the Zendesk queue if you got the following text added to the bottom: `Note: Logs Attached`
9. Repeat the test but don't add the logs
10. Check if the request on Zendesk does'nt have the note.

## Checklist
